Resources/Private/Partials/Filter/Group.html

```
@@ -0,0 +1,32 @@
+<f:if condition="{colConfig}">
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)